### ISSUE TITLE
Ability to Disable Font Anti-Aliasing?

### ISSUE THREAD

**Issue by TheEmeraldFalcon** [equalsraf/neovim-qt#989]
I'm not sure if there is a way to do this, and if there is, I can't seem to find any documentation about it, but there should be a way to disable font's anti-aliasing within in the editor.

In MacVim I believe you can do `:set noantialias`, and that will disable the anti-aliasing. Something like this would be really useful.

To me, this feels like something pretty basic, so my best guess is that it's already implemented, but I can't find anything about it.

**Comment by jgehrig** [comment]
> To me, this feels like something pretty basic, so my best guess is that it's already implemented, but I can't find anything about it.

Unfortunately, there is no option to disable font anti-aliasing in neovim-qt.

I don't have any plans to implement this feature since this is not a common request I have seen. The more options we add, the harder it is to maintain and test them all.

However, I'm not opposed to adding this feature if enough users request it.

This is the mechanism Qt uses to control font anti-aliasing:
https://doc.qt.io/qt-5/qfont.html#setStyleStrategy

**Comment by TheEmeraldFalcon** [comment]
> I don't have any plans to implement this feature since this is not a common request I have seen. The more options we add, the harder it is to maintain and test them all.

Fair enough, I think I'll attempt to add it myself, but I'm not great with C++ so I'm not sure how well it will go. Thanks for linking Qt's docs on the subject though, that'll definitely help.
